### DISCUSSION TITLE
Feature Suggestion

### DISCUSSION THREAD

**Discussion by Joester202** [sll552/DiscordBee#78] [Ideas]
Don't know if its possible, but if it is, I would love to see a feature where the image that shows up is the album cover.

**Comment by sll552** [comment]
This is one of the oldest requests and I have still not forgotten about it. It is tracked at #3 
I can say this much: its closer now than it was a month ago.

**Reply by Joester202** [reply]
Glad to hear it, keep up the good work my man!

**Reply by Svarr** [reply]
@sll552 Are you aware, that discord-rpc has been dropped?
> This library has been deprecated in favor of Discord's GameSDK. [Learn more here](https://discord.com/developers/docs/game-sdk/sdk-starter-guide)

(See https://github.com/discord/discord-rpc#readme)

According to the [old documentation](https://discord.com/developers/docs/rich-presence/how-to), the replacement to the RPC functionality was now provided by the [Activity Manager](https://discord.com/developers/docs/game-sdk/activities).

**Reply by sll552** [reply]
Yes, it was replaced in https://github.com/sll552/DiscordBee/commit/bc88a6549fddc68f1d646d7363315877ce33da5f which was released with v1.3.0 of the plugin

**Reply by Svarr** [reply]
I see now. Because you didn't use the GameSDK I assumed you were still using the old DiscordRPC, not an unofficial NuGet Package.

**Reply by sll552** [reply]
GameSDK seemed complicated and not well documented at that time, so I went with something I could easily integrate (because it was available as NuGet Package).

**Reply by Svarr** [reply]
Oh yes, that's sensible. I tried to implement it yesterday and only got it to crash MusicBee because `Discord.RunCallbacks()` tried to write to protected memory. 🤷

**Reply by sll552** [reply]
I guess thats because the plugin is built for "AnyCPU" and the GameSDK (Unfortunately GameSDK is not OSS, but I think its still native with a C# wrapper around it) would need to know the correct arch to load the correct dll.
But I am just guessing, as I don't really like to use closed source native binaries from some company that declared OSS failed for them (https://github.com/discord/discord-rpc/issues/290)
